setup/gcp/installer/forseti_installer.py

```diff
@@ -168,8 +194,9 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             deployment_tpl_output_path = (
                 constants.DEPLOYMENT_TEMPLATE_OUTPUT_PATH.format(bucket_name))
 
-            print('Copying {} to {}'.format(deployment_tpl_path,
-                                            deployment_tpl_output_path))
+            print('Copying the Forseti {} deployment template to:\n    {}'
```

Comment:
You can use the `\t` character to actually tab.

Reply:
Updated to use \t (8 spaces on gcloud console.)